presto-docs/src/main/sphinx/connector/hive-security.rst

```diff
@@ -54,13 +54,14 @@ Hive does not exactly follow the SQL standard, there are the following
 limitations and differences:
 
```

Comment:
Do you have any plans to implement `ALTER TABLE SET AUTHORIZATION`?

Reply:
I wasn't planning to, since it's not needed. Unlike with schemas, individual permissions can be granted on tables.
As discussed on the issue, it's preferable to stay with what's possible in Hive and not diverge from that (Hive cannot grant permissions on schemas, but it can change the owner of a schema).